keycloak/themes/hmda/login/login.ftl

```diff
@@ -11,11 +11,11 @@
                     <legend class="usa-drop_text">Sign in</legend>
                     <span>or <a href="${url.registrationUrl}">create an account</a></span>
 
-                    <label for="username"><#if !realm.registrationEmailAsUsername>${msg("usernameOrEmail")}<#else>${msg("email")}</#if></label>
+                    <label for="username"><#if !realm.loginWithEmailAllowed>${msg("username")}<#elseif !realm.registrationEmailAsUsername>${msg("usernameOrEmail")}<#else>${msg("email")}</#if></label>
                     <#if usernameEditDisabled??>
                         <input id="username" name="username" type="text" autocapitalize="off" autocorrect="off" value="${(login.username!'')?html}" disabled>
                     <#else>
-                        <input id="username" name="username" value="${(login.username!'')?html}" type="text" autofocus />
+                        <input id="username" name="username" value="${(login.username!'')?html}" type="text" autofocus autocomplete="off" />
```

Comment:
It's a bit strange.  It comes from https://github.com/keycloak/keycloak/commit/5c2bf7c822ec6a8fe72feb52872c452c34df100d, but the corresponding Github PR has no description, and the corresponding JIRA issue doesn't seem to be available to the public.

I can understand why they would make this change though.  Many sites don't allow username fields to be cached.